data_structures/linked-list/ll_merge.py

```diff
@@ -15,25 +15,3 @@ def merge_lists(list_one, list_two):
             list_two.insert_before(current_two.val, current.val)
```

Comment:
This only works if all the values in your linked list are unique, as `insert_before` inserts before the first correct value it finds.